lib/initSync.js

```diff
@@ -1,37 +1,38 @@
 var config = require('./config');
+var _ = require('lodash');
 var globalCollisionHandler = require('./data_collision_handlers/globalHandler');
+var hashFunctions = require('./hash-functions');
 
 var wfmSync = require('fh-wfm-sync/lib/server');
 
 /**
  *
- * Initialising sync handlers
+ * Initialising sync handlers for the data sets managed by sync.
  *
  * @param mediator
  * @param mbaasApi
  * @param useMongooseStore
  */
 module.exports = function initSync(mediator, mbaasApi, useMongooseStore) {
-  var syncOptions =   config.get("syncOptions");
+
+  var syncOptions = config.get("syncOptions");
   syncOptions.dataCollisionHandler = globalCollisionHandler;
 
-  /**
-   *
-   * We are using mongoose models that have a `updatedAt` field.
-   *
-   * This can be used to detect if the record has changed.
-   *
-   * This is only used for mongoose. Otherwise, the default hashing functions are assigned.
-   */
-  if (useMongooseStore) {
-    syncOptions.hashFunction = function generateHash(record) {
-      return record.updatedAt ? new Date(record.updatedAt).getTime() : undefined;
-    };
-  }
+  _.each(['workorders', 'result', 'workflows', 'messages'], function(dataSetId) {
```

Comment:
could we get those as constant list up top e.g. SYNC_DATASETS or move it to config?